pkg/operator/controller/ingress/deployment.go

```diff
@@ -523,6 +528,15 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		LoadBalancingAlgorithm string `json:"loadBalancingAlgorithm"`
 		DynamicConfigManager   string `json:"dynamicConfigManager"`
 		ContStats              string `json:"contStats"`
+		// maxDynamicServers specifies the number of dynamic server slots that will be added to each route.
+		// This setting can only be configured if dynamicConfigManager is `true`.
+		// The default value is 1, with a maximum of 10.
+		// Dynamic server slots help to avoid reloads during endpoint scale-ups. The more dynamic servers
+		// added, the fewer reloads required when scaling up.
+		// Note, however, that dynamic servers consume memory even when not enabled.
+		// Use this analysis of the memory usage to assess the impact of different numbers of servers:
+		// https://gist.github.com/frobware/2b527ce3f040797909eff482a4776e0b
+		MaxDynamicServers string `json:"maxDynamicServers"`
```

Comment:
This should probably be an int, not a string.  Users may be confused that they'd have to specify 10000 as "10000" here.

Reply:
Well, I just followed the same pattern as for the other overrides. Two of them (`dynamicConfigManager`,`contStats`) could be `bool`s because `bool` is a valid JSON type.

Reply:
True, but I think we have precedent for string-based bools.  Choosing the string type for numbers allows malformatting, like "100K", which will fail later.